lib/ibandit/local_details_cleaner.rb

```diff
@@ -60,6 +61,13 @@ def self.clean_at_details(local_details)
       }
     end
 
+    def self.clean_au_details(local_details)
+      {
+        branch_code:    local_details[:branch_code].delete("-"),
```

Comment:
Should we clean spaces here too?

Reply:
So we don't do this for any other country. I don't think we need this check here.